rsql-jpa/src/test/java/io/github/perplexhub/rsql/RSQLJPASupportPostgresJsonTest.java

```diff
@@ -602,6 +603,24 @@ private static Stream<Arguments> doesNotConvertInteger() {
         ).filter(Objects::nonNull);
     }
 
+    private static Stream<Arguments> sortByText() {
+        var e1 = new PostgresJsonEntity(Map.of("a", "abc", "b", 1));
```

Comment:
Try to mix string, numbers, booleans and null in a third key to ensure we never raise an exception.

Reply:
I dont get that comment, you're expecting a "c" key to mix data types? But thats not related to "sorting by text". Maybe  we should add another test with mixed types?

Reply:
Something simple as

```
var e2 = new PostgresJsonEntity(Map.of("a", "ABC", "b", 2, "c", 1));
var e3 = new PostgresJsonEntity(Map.of("a", "DEF", "b", 3, "c", "a"));
var e4 = new PostgresJsonEntity(Map.of("a", "GHI", "b", 4, "c", true));
var e5 = new PostgresJsonEntity(Map.of("a", "ghi", "b", 5, "c", null));
var e6 = new PostgresJsonEntity(Map.of("a", "klm", "b", 6));
```

In fact, we don't care about the result of sorting with `c`, it's just to ensure it does not throw any exception.

Reply:
Là c’est sûr qu'on se prend une NPE sur cette ligne :
`var e5 = new PostgresJsonEntity(Map.of("a", "ghi", "b", 5, "c", null));`

Mais ok, je vois l’idée, je ferai ça lundi